Cargo.toml

```diff
@@ -19,6 +19,9 @@ chain-core = { path = "./chain-libs/chain-core" }
 serde = { version = "1.0.93", features = ["derive"] }
 bech32 = "0.6"
 js-sys = "0.3.24"
+rand = { version = "0.6", features = ["wasm-bindgen"] }
```

Comment:
use `rand_os` directly instead of using OsRng re-exported from `rand`

Reply:
done